### PR TITLE
Adding publication

### PR DESCRIPTION
**Pull request by ddooley**

### CODE REVIEW

File: ontology/foodon.md [hunk 1]
```diff
@@ -35,6 +35,9 @@ tracker: https://github.com/FoodOntology/foodon/issues/
 activity_status: active
 repository: https://github.com/FoodOntology/foodon
 preferredPrefix: FOODON
+publications: 
+   - id: https://doi.org/10.1038/s41538-018-0032-6
```
Comment (cthoyt) [inline ontology/foodon.md:39]:
```suggestion
  - id:  https://www.ncbi.nlm.nih.gov/pubmed/31304272
```

File: ontology/foodon.md [hunk 2]
```diff
@@ -35,6 +35,9 @@ tracker: https://github.com/FoodOntology/foodon/issues/
...
+     title: "FoodOn: a harmonized food ontology to increase global food traceability, quality control and data integration"
```
Comment (cthoyt) [inline ontology/foodon.md:40]:
```suggestion
    title: "FoodOn: a harmonized food ontology to increase global food traceability, quality control and data integration"
```

Comment (ddooley) [reply ontology/foodon.md:40]: Thanks for catching that.

**Comment by cthoyt** [comment]
References https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1826

**Pull request merged**